auction-server/src/state.rs

```diff
@@ -297,6 +302,7 @@ pub struct BidStatusWithId {
 
 #[derive(Clone)]
 pub struct ExpressRelaySvm {
```

Comment:
feels weird that this is stored separately from the `chains_svm` info. maybe it should be combined with that somehow? e.g. `chains_svm_info: SvmInfo`, with `SvmInfo` containing both the ExpressRelaySvm stuff and the hash map to specific chain information?

Reply:
realize it's not related to this PR, but just something i'm noticing now

Reply:
I guess the main reason is these fields are shared across different svm chains. Hopefully we can remove this struct soon alltogether